packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -137,15 +138,23 @@ private void prepareMediaRecorder(String outputFilePath) throws IOException {
   }
 
   @SuppressLint("MissingPermission")
-  public void open() throws CameraAccessException {
+  public void open(String imageFormatGroup) throws CameraAccessException {
+    HashMap<String, Integer> imageFormatMap = new HashMap<>();
+    // Current supported outputs
+    imageFormatMap.put("yuv420", 35);
+    imageFormatMap.put("jpeg", 256);
```

Comment:
This can be made a static member so it doesn't have to be initialised every time the open method is called. Also I think a better name would be `supportedImageFormats`:

```java
    // Declaring the static map 
    private static Map<String, integer> supportedImageFormats 
        = new HashMap<>() { 
              supportedImageFormats.put("yuv420", 35);
              supportedImageFormats.put("jpeg", 256);
          }